pkgs/tools/networking/tran/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "tran";
+  version = "0.1.43";
+
+  src = fetchFromGitHub {
+    owner = "abdfnx";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-qp4g1ZLRIIz0CZ/Zey354g0j9ePE4pGb82IivLezU7s=";
+  };
+
+  vendorSha256 = "sha256-JmRTI5ZBSFULfI+ki3hI8TPaS6IVP9D14r4DwK/nx1Y=";
+
+  subPackages = [ "." ];
+
```

Comment:
The version is not set in the output

```bash
[nix-shell:~/.cache/nixpkgs-review/pr-196252]$ ./results/tran/bin/tran version
tran version  
```

Could you try to set the version through [LDFLAGS](https://github.com/abdfnx/tran/blob/main/Makefile#L8)? Something like the example below could help.

```bash
  ldflags = [
    "-w"
    "-s"
    "-X main.version=v${version}"
  ];
```

Reply:
Thanks. Somehow missed it. Force-pushed with this.